images/pulp/Containerfile

```
@@ -0,0 +1,16 @@
+ARG PULPCORE_VERSION=nightly
+
+FROM quay.io/centos/centos:stream9
+
+ARG PULPCORE_VERSION
+
+RUN dnf -y update && \
+    dnf -y --nodocs --setopt install_weak_deps=False install \
+    https://yum.theforeman.org/pulpcore/${PULPCORE_VERSION}/el9/x86_64/pulpcore-repos-${PULPCORE_VERSION}-0.1.nightly.el9.noarch.rpm && \
```

After https://github.com/theforeman/foreman-infra/pull/2266 is merged I'd expect this to work:
```suggestion
    https://yum.theforeman.org/pulpcore/${PULPCORE_VERSION}/el9/x86_64/pulpcore-repo-latest.el9.noarch.rpm && \
```